### DISCUSSION TITLE
Cannot see Statistic Channel

### DISCUSSION THREAD

**Discussion by Thorn2910** [nobl/ioBroker.senec#508] [Q&A] [answered]
Only BMS / ENERGY / SYS_Update / WIZARD and info Channels available

**Comment by nobl** [comment]
Unfortunately this is not a bug but something that SENEC removed from the API.
I am currently working on a version that pulls ALL information available via the API - but of course statistics aren't available anymore.

The only way to still get access to statistics is to request a downgrade from senec or to ask them (if one is still on an older release) to not update without explicit consent granted (which is possible via support).
(or maybe buy their €999 system (corporation with another vendor) they announced a while ago)

I expect the new release on "latest" to be available before end of april. Just conducting internal testing.

**Comment by Thorn2910** [comment]
Thank you!

**Comment by nobl** [comment]
New release is published (should be available on the latest-branch on iobroker tomorrow).
While I cannot fix the removed statistics it now pulls many other available states.

If there are unknown states you will see info messages in the logfile. I am highly interested in them so I can update the library dealing with them in case they need special handling.

**Comment by nobl** [comment, marked as answer]
Update: https://github.com/nobl/ioBroker.senec/issues/6#issuecomment-830825152
